### PR TITLE
Updating readme with local dev instructions

### PR DESCRIPTION
**Pull request by philipdouble**

### CODE REVIEW

**Comment by cognizen** [comment]
@shaheinm Please add more details to the end regarding populating strapi - maybe that's its own section actually? Adding this from notes I took yesterday to help begin thinking about pass offs and on-boarding...

File: README.md [hunk 1]
```diff
@@ -49,24 +54,91 @@ To learn React, check out the [React documentation](https://reactjs.org/).
...
+To run the system locally a developer must setup and install 4 additional applications:
+
+- docker
+- strapi-mongo
+- opl-content-api
+- opl-api-gateway
+
+### Install Docker
+
+Goto: <https://docs.docker.com/get-docker/> and install the appropriate version of Docker for your system.
```
Comment (shaheinm) [inline README.md:90]: ...or, preferably, [Podman](https://podman.io/getting-started/installation)! :stuck_out_tongue_winking_eye:

File: README.md [hunk 2]
```diff
@@ -49,24 +54,91 @@ To learn React, check out the [React documentation](https://reactjs.org/).
...
+`git clone https://github.com/openpracticelibrary/opl-content-api.git`
+
+Then cd into the new project:
+
+`cd opl-content-api`
+
+We need to install the dependencies and then update one of these dependencies and then re-install it:
+
+`npm i`
+
```
Comment (shaheinm) [inline README.md:113]: I've removed the `postinstall` step in my latest updates (CI doesn't like it), so you'll need to run `npm run patch` after install.  Can still one line it with `npm install && npm run patch`

Comment (shaheinm) [reply README.md:113]: Should probably include a link to the Strapi PR, as well, to let people know why they have to go through all this painful mess.

File: README.md [hunk 3]
```diff
@@ -49,24 +54,91 @@ To learn React, check out the [React documentation](https://reactjs.org/).
...
+`"@apollo-federation": "^0.15.0",`
+
+and then after saving the file run an npm install again:
+`npm i`
+
+Then start the server:
+
+`npm run develop`
+
+### Install opl-api-gateway
```
Comment (shaheinm) [inline README.md:125]: You can optionally do this. You could also just connect the frontend directly to Strapi (opl-content-api) at port `:1337`

File: README.md (hunk 1 above)
Comment (philipdouble) [reply README.md:90]: @shaheinm Your directions - this is just what I transcribed from your demo - update it with instructions please...

File: README.md (hunk 2 above)
Comment (philipdouble) [reply README.md:113]: @shaheinm see above :)

**Pull request closed without merging**